frontend/src/resources/pod.ts

```diff
@@ -80,10 +80,22 @@ export async function getHivePod(namespace: string, name: string, status: string
   /* istanbul ignore else */
   if (status === ClusterStatus.creating) {
     hiveJob = await getProvisionPod(namespace, name)
-  } else if (status === ClusterStatus.provisionfailed || status === ClusterStatus.deprovisionfailed) {
+  } else if (
+    status === ClusterStatus.provisionfailed ||
+    status === ClusterStatus.deprovisionfailed ||
+    status === ClusterStatus.posthookjob ||
+    status === ClusterStatus.posthookfailed
```

Comment:
I noticed that when posthook runs or fails we can't access the provision logs from the details page because these cases weren't here. The link component still renders however, it just goes nowhere. It made more sense to me to add these cases.